src/components/ProcessForm/ProcessForm.tsx

```diff
@@ -0,0 +1,260 @@
+                            isInvalid={validation && validation.BuyerError !== ""}
+                        />
+                        <Form.Control.Feedback type="invalid">
+                            {validation ? validation.BuyerError : ""}
+                        </Form.Control.Feedback>
+                    </Col>
+                    <Col xl="6">
+                        <Form.Label className="required"><strong>Contracting Officer</strong></Form.Label>
+                    </Col>
+                    <Col xl="6">
+                        <Form.Control
+                            as={PeoplePicker}
+                            updatePeople={(p: IPerson[]) => {
+                                let persona = p[0];
+                                setProcess({ ...process, ContractingOfficer: persona ? new Person(persona) : new Person() });
+                            }}
+                            required
+                            isInvalid={validation && validation.ContractingOfficerError !== ""}
+                        />
+                        <Form.Control.Feedback type="invalid">
+                            {validation ? validation.ContractingOfficerError : ""}
+                        </Form.Control.Feedback>
+                    </Col>
+                    <Col xl="6">
+                        <Form.Label className="required"><strong>Small Business Professional</strong></Form.Label>
+                    </Col>
+                    <Col xl="6">
+                        <Form.Control
+                            as={PeoplePicker}
+                            updatePeople={(p: IPerson[]) => {
+                                let persona = p[0];
+                                setProcess({ ...process, SmallBusinessProfessional: persona ? new Person(persona) : new Person() });
+                            }}
+                            required
+                            isInvalid={validation && validation.SmallBusinessProfessionalError !== ""}
+                        />
+                        <Form.Control.Feedback type="invalid">
+                            {validation ? validation.SmallBusinessProfessionalError : ""}
+                        </Form.Control.Feedback>
+                    </Col>
+                    <Col xl="6">
+                        <Form.Label className="required"><strong>PoP (months, including options)</strong></Form.Label>
+                        <InfoTooltip id="pop">Agrees with DD2579 Block 8</InfoTooltip>
+                    </Col>
+                    <Col xl="6">
+                        <Form.Control
+                            type="number"
+                            value={process.SboDuration ? process.SboDuration : undefined}
+                            onChange={e => setProcess({ ...process, SboDuration: parseInt(getNumbersOnly(e.target.value)) })}
+                            isInvalid={validation && validation.SboDurationError !== ""}
+                        />
```

Comment:
I believe you can set minimum and maximum value properties (of some sort) on this

Reply:
Do you know what those limits should be? I can ask Kelly if not

Reply:
Based on the email from Kelly I guess it's 1 to infinity